.fixtures.yml

```diff
@@ -1,5 +1,12 @@
 fixtures:
   repositories:
+    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
```

Comment:
why do you add the modules?

Reply:
That was a mistake from the merge conflicts. Should be fixed